docs/frontend/installation/README.md

```diff
@@ -0,0 +1,106 @@
+## Prerequisites
```

Comment:
Never move ReadMe into docs folder. Please read the issue again

Reply:
Never moved it, kept a relevant part of the documentation there, named it `README.md`, so that it's the first thing that appears when you open `/doc/backend/installation/` on the web.

Reply:
No don't do it this way, Create separate docs folder for `Frontend` and `Backend`. This can be really confusing at times

Reply:
docs/frontend is for frontend
docs/backend is for backend